### PR TITLE
Portfolio: Desktop Version

### PR DESCRIPTION
**Pull request by batoorsohail**
# Portfolio: Desktop Version

In this project we added:
- Header
- Works
- About me
- Contact me

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -342,3 +360,174 @@ button:hover {
...
+    font-weight: 500;
+    font-size: 15px;
+    color: #344563;
+  }
+
+  .background {
+    background-image: url(./images/Header\ bg@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
```
Comment (mckent05) [inline style.css:400]: - Good job using media queries properly 👏 . But there seem to be a few differences in this section of your design from what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- The background image for this section is not properly aligned, please reference the marked images in the design template I attached below.
werewv

Figma Design:
rvevev

File: index.html [hunk 2]
```diff
@@ -1,183 +1,283 @@
...
+              ></a>
+            </li>
+          </ul>
+          <a href="#" class="button">Get my resume</a>
+      </div>
+      <div class="lagu">
+        <ul class="about-me">
+           <div class="down">
+            <a href="#">Languages</a>
+            <i class="fa-solid fa-chevron-right"></i>
```
Comment (mckent05) [inline index.html:206]: - You have used the wrong arrow icon in this section, please fix this 🙏 .Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
etnetnr

Figma template:
baetas

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @batoorsohail

# STATUS: CHANGES REQUIRED ✔️ ✔️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- The mobile and desktop version of your project looks impressive 👌 
- Your PR is professional 👏 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -1,183 +1,284 @@
...
+        <div class="web-name">Sohail</div>
+        <ul class="menu">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <a href="#" class="bar-button">
+          <i class="fa-solid fa-bars"></i>
+        </a>
+      </nav>
```
Comment (Nyame-Wolf) [inline index.html:26]: [OPTIONAl] I suggest you fix your mobile design to match figma template. The desktop version should not affect the mobile design but should complement each other. Currently, the mobile design, the background images are overriding the test in mobile version
Screenshot 2022-12-08 at 15 34 39
Please fix this.

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @batoorsohail  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- Linters are passing
- Correct use of gitflow
- Made changes requested by previous reviewer

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**